docs/data/material/customization/dark-mode/dark-mode.md

```diff
@@ -36,86 +36,62 @@ Adding `<CssBaseline />` inside of the `<ThemeProvider>` component will also ena
-        setMode((prevMode: PaletteMode) =>
-          prevMode === 'light' ? 'dark' : 'light',
-        );
-      },
-    }),
-    [],
-  );
+## System preference
 
-  // Update the theme only if the mode changes
-  const theme = React.useMemo(() => createTheme(getDesignTokens(mode)), [mode]);
+Some users sets a preference for light or dark mode through their operation system—either systemwide, or for individual user agents.
+The following sections explain how to apply these preferences to an app's theme.
 
-  return (
-    <ColorModeContext.Provider value={colorMode}>
-      <ThemeProvider theme={theme}>
-        <Page />
-      </ThemeProvider>
-    </ColorModeContext.Provider>
-  );
-}
-```
+### Built-in support
 
-Here is a fully working example:
+Use the `colorSchemes` node to build an application with multiple color schemes.
+The built-in color schemes are `light` and `dark` which can be enabled by setting the value to `true`.
 
-{{"demo": "DarkThemeWithCustomPalette.js", "defaultCodeOpen": false}}
+The light color scheme is enabled by default, so you only need to set the dark color scheme:
 
-## Toggling color mode
+```js
+import { ThemeProvider, createTheme } from '@mui/material/styles';
 
-To give your users a way to toggle between modes, you can add React's context to a button's `onClick` event, as shown in the following demo:
+const theme = createTheme({
+  colorSchemes: {
+    dark: true,
+  },
+});
 
-{{"demo": "ToggleColorMode.js", "defaultCodeOpen": false}}
+function App() {
+  return <ThemeProvider theme={theme}>...</ThemeProvider>;
+}
+```
 
-## System preference
+When `colorSchemes` is provided, the following features are activated:
+
+- Automatic switching between light and dark color schemes based on the user's preference.
+- Synchronization between window tabs—changes to the color scheme in one tab are applied to all other tabs
+- An option to [disable transitions](#disable-transitions) when the color scheme changes
+
+:::success
+To test the system preference feature, follow the guide on [emulating the CSS media feature `prefers-color-scheme`](https://developer.chrome.com/docs/devtools/rendering/emulate-css#emulate_css_media_feature_prefers-color-scheme).
+:::
 
-Users might have a preference for light or dark mode that they've set through their operating system—either systemwide, or for a single user agent.
+### Manual implementation
 
-You can make use of this preference with the [useMediaQuery](/material-ui/react-use-media-query/) hook and the [prefers-color-scheme](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) media query.
+You can make use of this preference with the [`useMediaQuery`](/material-ui/react-use-media-query/) hook and the [`prefers-color-scheme`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) media query.
 
 The following demo shows how to enable dark mode automatically by checking for the user's preference in their OS or browser settings:
 
```

Comment:
The example below about `useMediaQuery('(prefers-color-scheme: dark)')` might be confusing as it goes back to using `palette` for dark mode. What do you think about simplifying it:

```jsx
import * as React from 'react';
import useMediaQuery from '@mui/material/useMediaQuery';

function App() {
  const prefersDarkMode = useMediaQuery('(prefers-color-scheme: dark)');

  return (
    <div>
        prefersDarkMode: {prefersDarkMode.toString()}
    </div>
  );
}
```

If so, I would change the title to "Accessing prefers-color-scheme value"

Reply:
Thanks for the suggestion. Applied it.